kai/report.py

```diff
@@ -5,7 +5,8 @@
 from io import StringIO
 
 import yaml
-from kai_logging import KAI_LOG
+
+from kai.kai_logging import KAI_LOG
```

Comment:
I needed to tweak a few of the kai_logging imports, was seeing below when trying to run


```
$ time ./run_demo.py           
Traceback (most recent call last):
  File "/Users/jmatthews/git/jwmatthews/kai/example/./run_demo.py", line 12, in <module>
    from kai import Report
  File "/Users/jmatthews/git/jwmatthews/kai/example/../../kai/kai/__init__.py", line 7, in <module>
    from kai.incident_store import IncidentStore
  File "/Users/jmatthews/git/jwmatthews/kai/example/../../kai/kai/incident_store.py", line 11, in <module>
    from kai.report import Report
  File "/Users/jmatthews/git/jwmatthews/kai/example/../../kai/kai/report.py", line 8, in <module>
    from kai_logging import KAI_LOG
ModuleNotFoundError: No module named 'kai_logging'
```